.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
     "editor.formatOnSave": true,
     "rust-client.channel": "nightly-2021-03-19",
-}
+}
```

Comment:
Don't change this file

Reply:
My editor deleted the last empty line. I put back the original file.